rdrf/rdrf/views/lookup_views.py

```diff
@@ -42,97 +40,79 @@ def get(self, request, reg_code):
                     Q(working_groups__in=working_groups)
 
                 for patient_model in Patient.objects.filter(query):
-                    if patient_model.active:
-                        name = "%s" % patient_model
-                        results.append({"value": patient_model.pk, "label": name,
-                                        "class": "Patient", "pk": patient_model.pk})
+                    try:
+                        security_check_user_patient(self.request.user, patient_model)
```

Comment:
Does this mean that patients can only look themselves up? Will this break questionnaires functionality if it's enabled?

Reply:
You're right. I reverted this change as after looking more closely I found out that this view need to be restricted to staff members only.